es6/README.md

```diff
@@ -51,6 +51,35 @@ globals:
 The boolean value indicates whether the global is to be treated as
 read-only (`false`) or read-write (`true`).
 
+## Naming
+
+Classes and constructor functions should be TitleCased
```

Comment:
So one thing that violates this now is [`mobify-debug-js`](https://github.com/mobify/mobify-debug-js/blob/master/index.js#L19).

I _think_ that should become a lowercase function name... thoughts?